### DISCUSSION TITLE
Paint

### DISCUSSION THREAD

**Discussion by BOTkirial** [huxingyi/dust3d#137] [General]
Hey
I've been using dust for a while and it's awesome.

I just can't figure out how to paint a model ? I guess it's possible because of the video on Twitter with a banana being painted

When I use the brush tool it inflates the model where I use it

Any help ? Thanks

**Comment by huxingyi** [comment]
Hi BOTkirial, the painting feature is an experiment not released yet, I am pretty sure the modeling feature will be enhanced and focused on next release. For painting feature, not sure yet. Thanks.
